src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheckTest.java

```diff
@@ -102,7 +104,7 @@ public void testIt()
             "156:21: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, ":"),
             "262:14: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "}"),
         };
-        verify(checkConfig, getPath("InputWhitespace.java"), expected);
+        verify(checkConfig, getPath("InputWhitespaceAroundWhitespace.java"), expected);
```

Comment:
Hm. Whitespace around whitespace... Strange meaning. Maybe another postfix, as we already have "InputWhitespaceAround"?